src/pages/GapsPage.tsx

```diff
@@ -61,7 +87,12 @@ const GapsPage = () => {
       }
       setGapsIsLoading(true)
       getGapsAsync(moment(timestamp), operatorId, selectedRoute.lineRef)
-        .then(setGaps)
+        .then((gaps) => {
+          setGaps(gaps)
+          const ridesInTime = gaps?.filter((gap) => formatStatus([], gap) === TEXTS.ride_as_planned)
```

Comment:
Let's keep the formatting separated from logic. In the future, the formatting function may wrap the text and apply various modifications, so we probably don't want it to be coupled with this filter